### DISCUSSION TITLE
How can i load sass variable from JSON ?

### DISCUSSION THREAD

**Discussion by susanta96** [vercel/next.js#13545] [Help] [answered]
There are few packages out there who claim to compile **JSON** into sass variable , but everyone depends on node-sass. That's not good i don't wanna use previous node sass and next-sass . I wanna use the new built in sass module using dart-sass. Please help me out here, is there any package or next config can we do to make this possible ?

**Comment by greyGroot** [comment, marked as answer]
@susanta96 check this discussion: https://github.com/vercel/next.js/discussions/19672
